content/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository.md

```diff
@@ -14,6 +14,13 @@ versions:
   ghec: '*'
 shortTitle: Splitting a subfolder
 ---
+ 
+{% note %}
+
+ **Note:** You need Git version 2.22.0 or later to follow these instructions, otherwise `git filter-repo` will not work.
```

Comment:
```suggestion
**Note:** You need Git version 2.22.0 or later to follow these instructions, otherwise `git filter-repo` will not work.
```

Sorry, not sure how I only deleted one indentation, just sorting 👍